src/main/java/com/blazemeter/ciworkflow/CiPostProcess.java

```diff
@@ -283,22 +285,33 @@ protected JSONObject getFunctionalReport(Master master) {
     }
 
     public File makeReportDir(String reportDir) throws Exception {
-        File f = new File(reportDir);
+        File f;
```

Comment:
Could you simplify this part with [ternary operator](https://alvinalexander.com/java/edu/pj/pj010018)?